contrib/netbox.service

```diff
@@ -0,0 +1,24 @@
+[Unit]
+Description=Netbox WSGI
+Documentation=https://netbox.readthedocs.io/en/stable/
+After=network-online.target
+Wants=network-online.target
+
+[Service]
+Type=simple
+
+EnvironmentFile=/etc/sysconfig/netbox.env
+
+User=www-data
+Group=www-data
+PIDFile=${PidPath}
+WorkingDirectory=${WorkingDirectory}
+
+ExecStart=/usr/local/bin/gunicorn --pid ${PidPath} --pythonpath ${WorkingDirectory}/netbox --config ${ConfigPath} netbox.wsgi
```

Comment:
Set `--error-logfile` here if you want it to depend on WorkingDirectory: e.g. `${WorkingDirectory}/logs/netbox.log`

(aside: `logs` subdir means it can be made writeable to the gunicorn user`

Reply:
Someone previously complained about not using a gunicorn config file.  I can't win, so for now it will probably stick as is.